### PR TITLE
Add release notes for email connector 1.1.4

### PR DESCRIPTION
**Pull request by SebaElizalde**

### CODE REVIEW

File: modules/ROOT/pages/connector/connector-email.adoc [hunk 1]
```diff
@@ -5,6 +5,14 @@ endif::[]
 
 Support Category: Select
 
+== 1.1.4
+
+*April 22, 2019*
+
```
Comment (mcherem) [inline modules/ROOT/pages/connector/connector-email.adoc]: Can you confirm that the following are correct for this release? If so, this content would need to be added under the date for this release note. This is from the latest release note template.
[%header%autowidth.spread]
|===
|Software |Version
|Mule Runtime |4.1 and later
|Anypoint Studio |7.1 and later
|JDK |8
|===

Comment (george-hoffer) [reply modules/ROOT/pages/connector/connector-email.adoc]: We assume that what the devs provide in the release notes are correct. They test using the info provided. But please add "=== Compatibility" as the section above the compatibility info table.

File: modules/ROOT/pages/connector/connector-email.adoc [hunk 2]
```diff
@@ -5,6 +5,24 @@ endif::[]
...
+|===
+|Software |Version
+|Mule Runtime |4.1.1 and later
+|Anypoint Studio |7.1.1 and later
+|JDK |8
+|===
+
+=== Fixed Issues
+
+* Fixed an issue that caused NullPointerException when polling a mailbox folder. (https://www.mulesoft.org/jira/browse/MULE-16746[MULE-16746])
```
Comment (george-hoffer) [inline modules/ROOT/pages/connector/connector-email.adoc]: Add "a" before NullPointerException and surround that word with tick marks

File: modules/ROOT/pages/connector/connector-email.adoc (hunk 1 above)
Comment (mcherem) [reply modules/ROOT/pages/connector/connector-email.adoc]: Got it, thanks. Will add the table in the appropriate section per the template.

**Pull request merged**